scripts/tns_run.py

```diff
@@ -0,0 +1,20 @@
+import os
+from stream2hop import Utils as ut
+import json
+
+region = "us-west-2"
+cred_secret_name = "dev-gcn2hop-hopcreds"
+tns_api_key_secret_name = "tns_api_key"
+configDir = "/root/share"
+Location = "%s/kafkacat.conf" % configDir
+
+
+os.system("mkdir -p %s" % configDir)
+creds = ut.get_secret(cred_secret_name)
+if creds is not None:
```

Comment:
What if it _is_ None? Should you crash, or what?

Reply:
I moved the running line into the if statement to avoid crashes and if none, then print an error message.

Reply:
How about switching them around, so there is a shorter block under indentation? That way, you don't need to scan as far to understand the two branches:

```
if creds is None:
    print("creds cant be none, ya dummy")
    sys.exit(1)
creds_json = json.loads(creds)
// ... etc 
```